pudb/debugger.py

```diff
@@ -612,12 +584,11 @@ def __init__(self, func_name, code):
         self.code = code
 
     def __eq__(self, other):
-        return (
-                type(self) == type(other)
-                and
-                self.function_name == other.function_name
-                and
-                self.code is other.code)
+        return all([
+            type(self) == type(other),
+            self.function_name == other.function_name,
+            self.code is other.code,
+        ])
```

Comment:
This is weird to me. It incurs an extra function call penalty *and* it's less readable (to me).